contrib/lib/src/databases.rs

```diff
@@ -210,3 +215,19 @@ impl Poolable for redis::Connection {
             .map_err(DbError::PoolError)
     }
 }
+
+#[cfg(feature = "sqlite_pool")]
+impl Poolable for rusqlite::Connection {
+    type Manager = r2d2_sqlite::SqliteConnectionManager;
+    type Error = r2d2::Error;
+
+    fn pool(config: DatabaseConfig) -> Result<r2d2::Pool<Self::Manager>, Self::Error> {
+        let manager = if config.url == ":memory:" {
```

Comment:
Is this check necessary? I think I looked into it and don't remember exactly, but I believe rusqlite will pass ":memory:" unchanged to the sqlite engine, which handles that itself.

Reply:
I wasn’t sure if this was necessary. Let me review `rusqlite` and adjust it as necessary. I’d be surprised if it didn’t handle that as a URL correctly.